tensorhive/app/web/dev/src/components/views/ReservationsOverview.vue

```diff
@@ -162,7 +162,6 @@ export default {
         }
         this.parsedNodes.push(tempNode)
       }
-      this.loadCalendar()
```

Comment:
I had to delete this, because parseNodes in MySchedule will call this indirectly by emiting 'loadResources' event. Calling this twice won't work because of the wicked updateCalendar = !updateCalendar logic
@filschod does it seem legit?